docusaurus/docs/reactnative/basics/translations.mdx

```diff
@@ -1,48 +1,47 @@
 ---
 id: translations
 sidebar_position: 4
-title: Translations / i18n
+title: Adding Internationalization (i18n)
```

Comment:
Opted to call the page Internationalization to match the name of the Class we use for this (`Streami18n`).